web/packages/plugins/dmt-app/src/pages/SearchPage.tsx

```diff
@@ -229,7 +230,7 @@ function FilterContainer({
 }: {
   search: (query: any) => void
   queryError: string
-  searchFilter: any
+  searchFilter: TGenericObject
   setSearchFilter: (newFilter: any) => void
```

Comment:
```suggestion
  setSearchFilter: (newFilter: any) => void
```
Can we specify type here?

Reply:
type added in https://github.com/equinor/data-modelling-tool/pull/1381